examples/slave/nuttx/xmc4800/od_populator.cc

```diff
@@ -0,0 +1,89 @@
+/// This file is auto generated by od_generator.
+
+#include "kickcat/od_populator.h"
+#include "kickcat/CoE/OD.h"
+
+namespace kickcat
+{
+    void populateOD()
+    {
+        {
+            static CoE::Object object
+            {
+                .index = 0x1018,
+                .code = CoE::ObjectCode::RECORD,
+                .name = "Identity Object"
+            };
+            CoE::addEntry(object,0,8,7,static_cast<CoE::DataType>(5),"Subindex 000",0x4);
+            CoE::addEntry(object,1,32,7,static_cast<CoE::DataType>(7),"Vendor ID",0x29c);
+            CoE::addEntry(object,2,32,7,static_cast<CoE::DataType>(7),"Product code",0x32);
+            CoE::addEntry(object,3,32,7,static_cast<CoE::DataType>(7),"Revision number",0x0);
+            CoE::addEntry(object,4,32,7,static_cast<CoE::DataType>(7),"Serial number",0xcadefeca);
```

Comment:
Strange, the serial number is different from the foot.xml.
has it just been generated on a different xml ?

Reply:
In the ESI, the default data is written in little endian. That's why